### PR TITLE
 jump_search.cpp

### PR DESCRIPTION
**Pull request by rakesh0x7**
This PR for issue #162 code for jump search in c++

### CODE REVIEW

File: Search_Techniques/Jump_Search/jump_search.cpp [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+   }
+   return -1;
+}
+
+int main() {
+   int n, item, loc;
+   cout << "\n Enter number of items: ";
+   cin >> n;
+   int arr[n]; //creating an array of size n
+   cout << "\n Enter items: ";
```
Comment (ankitapuri) [inline Search_Techniques/Jump_Search/jump_search.cpp:30]: mention it works just for sorted arrays

Comment (rakesh0x7) [reply Search_Techniques/Jump_Search/jump_search.cpp:30]: you're saying about adding comment in the code?

Comment (ankitapuri) [reply Search_Techniques/Jump_Search/jump_search.cpp:30]: yes for better understanding for the ones who don't know about it .

Comment (rakesh0x7) [reply Search_Techniques/Jump_Search/jump_search.cpp:30]: Haa Done!!
Thank you for correcting my mistakes

**Review by ankitapuri — APPROVED** [review]

**Pull request merged**